src/sentry/static/sentry/app/components/charts/components/tooltip.jsx

```diff
@@ -38,14 +38,26 @@ function getFormatter({filter, isGroupedByDate, truncate, formatAxisLabel, utc})
     //
     // Otherwise, it will be an `Object`, and is a tooltip for a single item
     const isAxisItem = Array.isArray(seriesParamsOrParam);
+    const axisFormatterOrDefault = formatAxisLabel || defaultFormatAxisLabel;
+
+    // Special tooltip if component is a `markPoint`
+    if (!isAxisItem && seriesParamsOrParam.componentType === 'markPoint') {
+      const timestamp = seriesParamsOrParam.data.coord[0];
+      const label = axisFormatterOrDefault(timestamp, isGroupedByDate, utc);
+      return `<div>${label} - ${seriesParamsOrParam.name}</div>
```

Comment:
Do `label` or `seriesParamsOrParam.name` contain user input?

Reply:
Label is a formatted time stamp and name is a static string